stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_composer_default_center_content.xml

```diff
@@ -66,7 +66,6 @@
             android:background="@null"
             android:ellipsize="end"
             android:hint="@string/stream_ui_message_composer_hint_normal"
-            android:inputType="textCapSentences|textMultiLine"
```

Comment:
I typically prefer to keep the xml attributes in the new message composer, even if they are overridden programmatically. My train of thought below:

```
public class CustomMessageComposerCenterContent : FrameLayout, MessageComposerContent {

   // Copy-pasted layout with "android:inputType="textCapSentences|textMultiLine"
   private lateinit var binding: StreamUiMessageComposerCustomCenterContentBinding

    override fun attachContext(messageComposerContext: MessageComposerContext) {
        val style = messageComposerContext.style
        // Clients can use "style.messageInputInputType" here, but I expect 
        // that in 90% of the cases they will either rely on the defaults in the layout
        // or modify the copy pasted layout.
    }

    override fun renderState(state: MessageComposerState) {
        // ...
    }
}
```